lib/kino/table.ex

```diff
@@ -45,7 +49,7 @@ defmodule Kino.Table do
               {:ok,
                %{
                  columns: list(column()),
-                 rows: list(row()),
+                 data: columnar() | row_based(),
```

Comment:
We decided we don't want to send rows as field maps, because we repeat fields for every row. So in either case it's a list of lists, to distinguish they need to give us `{:rows, [[...], ...]}` or `{:columns, [[...], ...]}` (and we assume it matches the order of columns).